cppwinrt/cppwinrt.vcxproj

```diff
@@ -259,9 +259,6 @@
       <Command>
       </Command>
     </PostBuildEvent>
-    <Manifest>
-      <AdditionalManifestFiles>app.manifest;%(AdditionalManifestFiles)</AdditionalManifestFiles>
-    </Manifest>
```

Comment:
Why did this get dropped? This was to enable long path support.

Reply:
The manifest is already included in the project as a "Manifest file" type (https://github.com/microsoft/cppwinrt/blob/aa6467d8c6c1905f8b4321a4e08b7034e6445e40/cppwinrt/cppwinrt.vcxproj#L118), which from what I can tell means it's automatically used when linking. This can be verified in the build log as the `/manifestinput:app.manifest` flag passed to `link.exe`.

This extra `AdditionalManifestFiles` attribute is in a conditional "Debug|x64" block so it doesn't apply to other configurations anyway. It also makes lld-link produce a broken binary (filed https://github.com/llvm/llvm-project/issues/58444), which is the main reason for removing this. (Though I ended up not switching to lld-link anyway due to https://github.com/llvm/llvm-project/issues/58445. I guess this may be fixable in the project file but there's no rush to enable this.)